src/main/java/com/openlattice/mapstores/TestDataFactory.java

```diff
@@ -291,11 +306,11 @@ public static PropertyType dateTimePropertyType() {
                 EdmPrimitiveTypeKind.DateTimeOffset,
                 Optional.of( r.nextBoolean() ),
                 Optional.of( Analyzer.STANDARD ),
-                Optional.of( INDEX_TYPES[ r.nextInt( INDEX_TYPES.length ) ] ) );
+                Optional.of( INDEX_TYPES[r.nextInt( INDEX_TYPES.length )] ) );
     }
 
     public static PropertyType propertyType() {
-        return propertyType( INDEX_TYPES[ r.nextInt( INDEX_TYPES.length ) ] );
+        return propertyType( INDEX_TYPES[r.nextInt( INDEX_TYPES.length )] );
```

Comment:
there's enough of these `INDEX_TYPES[r.nextInt( INDEX_TYPES.length )]` that it seems maybe worth it to make a separate method for those

(same comment for analyzers)

Reply:
Will do... btw, what are our preferences on auto-format for these cases? 
`array[1]` or `array[ 1 ]`

Reply:
I like `array[ 1 ]` but I usually just default to whatever our auto-formatter does (for consistency / since it'll probably end up that way eventually once someone edits that file)